doc/SecretManagement.md

```diff
@@ -7,32 +7,47 @@ Kubeseal: Encrypts and decrypts secrets at k8s level. Kubeseal does not allow lo
+  * [SSH into your environment-specific node](./RUNBOOK.md#ssh)
+  * Run the following command to get the Kubeseal certificate:
+    ```bash
+    KUBECONFIG=/etc/rancher/k3s/k3s.yaml kubeseal --fetch-cert --controller-namespace sealed-secrets --controller-name=sealed-secrets
+    ```
+  * Save this to your local machine with a .pem extension (saved as `qa-sealedsecret.pem` in the below example)
+* Save the Vault password for k8s from the Simple password manager to your local machine (saved as `~/.vault_password_k8s` in the example below). This will be used to encrypt and decrypt secrets on your local machine.
 
 ## Update/edit a secret key
 * Step1: Decrypt Vault secrets
 * Step2: Edit the secret key
-* Step3: Encrypt Vault secrets
-* Step4: Encrypt Kubeseal secrets
+* Step3: Encrypt secrets with Vault
+* Step4: Encrypt secrets with Kubeseal
 * Step5: Commit changes
 
-Example: Updated the secret in qa/secrets/simple-server.sealedsecret.yaml
+**Example**: Update secrets in qa/secrets/simple-server.sealedsecret.yaml
 ```bash
 cd k8s/environments
 
+# This decrypts secrets with Ansible Vault and your Vault password
 ./decrypt ~/.vault_password_k8s qa/secrets/simple-server.sealedsecret.yaml.decrypted.vault
 
-# Edit the secret key in the file
-# Base64 encode secret value
-./encrypt ~/.vault_password_k8s qa/secrets/simple-server.sealedsecret.yaml
+# Edit secrets in the decrypted file `simple-server.sealedsecret.yaml.decrypted`.
+
+# Secrets in this file are base64-encoded because Kubeseal requires this.
+# Ensure your new secrets are encoded appropriately as well before adding to the file.
+# You can use the base64 utility on macos to do this.
 
+# This encrypts the updated secrets with Ansible Vault.
+./encrypt ~/.vault_password_k8s qa/secrets/simple-server.sealedsecret.yaml.decrypted
+
+# Encrypts updated secrets with Kubeseal
+# Here, the Kubeseal certificate is copied over from the qa jumpbox and saved locally as `qa-sealedsecret.pem`
 ./kubeseal_encrypt ~/.qa-sealedsecret.pem qa/secrets/simple-server.sealedsecret.yaml.decrypted
 
 # Commit the changes
 ```
 
-`decrypt` , `encrypt` and `kubeseal_encrypt` supports multiple options to decrypt/encrypt secrets in a all the environments or a specific environment or a specific file. Please refer the usage section for more details.
+`decrypt` , `encrypt` and `kubeseal_encrypt` supports multiple options to decrypt/encrypt secrets in all environments or a specific environment or a specific file. Please refer to the usage section for more details.
+
+## Secrets deployment
+
+The secrets file is auto-synced on [ArgoCD](https://argocd-qa.simple.org/applications/argocd/secrets?view=tree&resource=&node=bitnami.com%2FSealedSecret%2Fsimple-v1%2Fsimple-server%2F0) (find the login credentials on the Simple password manager). To deploy manually, directly update the secrets (don't forget to base64 encode them!) in the simple-server live manifest on ArgoCD and hit `sync`.
```

Comment:
To deploy manually, directly update the secrets (don't forget to base64 encode them!) in the simple-server live manifest on ArgoCD and hit `sync`.  is not recommended all changes should be in git

Reply:
@prabhanshuguptagit ^

Reply:
:shipit: